lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -49,7 +49,21 @@ def load_graph(url:)
         end
 
         def parse_search_authority_response
-          results = results_mapper_service.map_values(graph: @graph, predicate_map: preds_for_search, sort_key: :sort,
+          predicate_map = preds_for_search
+          ldpath_map = ldpaths_for_search
+
+          raise Qa::InvalidConfiguration, "do not specify results using both predicates and ldpath in search configuration for LOD authority #{authority_name} (ldpath is preferred)" if predicate_map.present? && ldpath_map.present? # rubocop:disable Metrics/LineLength
+          raise Qa::InvalidConfiguration, "must specify label_ldpath or label_predicate in search configuration for LOD authority #{authority_name} (label_ldpath is preferred)" unless ldpath_map.key?(:label) || predicate_map.key?(:label) # rubocop:disable Metrics/LineLength
```

Comment:
Moved exception checking up to this method since it has to check whether both ldpath and predicates before it can determine that the config is invalid.